sql/moz-fx-data-marketing-prod/ga_derived/www_site_metrics_summary_v2/metadata.yaml

```
@@ -0,0 +1,24 @@
+friendly_name: WWW Site Metrics Summary V2
+description: |-
+  Summary of site metrics from Google Analytics (specifically Google Analytics 4)
+owners:
+- kwindau@mozilla.com
+labels:
+  incremental: true
+  owner1: example
+scheduling:
+  dag_name: bqetl_google_analytics_derived_ga4
+bigquery:
+  time_partitioning:
+    type: day
+    field: date
+    require_partition_filter: true
```

```suggestion
    require_partition_filter: false
```

Aggregate tables don't usually need partition filters, since the cost to query is very low

---

Oh good point! Will change - thanks!